docs/embedding/sdk/questions.md

```diff
@@ -69,18 +69,19 @@ export default function App() {
+| isSaveEnabled      | `boolean`                                          | (optional) Whether people can save the question.                                                                                                                                                                                                                                                                   |
+| withResetButton    | `boolean`                                          | (optional, default: `true`) Determines whether a reset button is displayed. Only relevant when using the default layout                                                                                                                                                                                            |
+| withTitle          | `boolean`                                          | (optional, default: `false`) Determines whether the question title is displayed. Only relevant when using the default layout.                                                                                                                                                                                      |
+| customTitle        | `string \| undefined`                              | (optional) Allows a custom title to be displayed instead of the default question title. Only relevant when using the default layout.                                                                                                                                                                               |
+| onBeforeSave       | `() => void`                                       | (optional) A callback function that triggers before saving. Only relevant when `isSaveEnabled = true`.                                                                                                                                                                                                             |
+| onSave             | `() => void`                                       | (optional) A callback function that triggers when a user saves the question. Only relevant when `isSaveEnabled = true`.                                                                                                                                                                                            |
+| saveToCollectionId | `number`                                           | (optional) The target collection to save the question to. This will hide the collection picker from the save modal.                                                                                                                                                                                                |
```

Comment:
The diffs is a bit dramatic here - the prop name `saveToCollectionId` is long so the table gets expanded.

Reply:
The "Hide whitespace changes" thankfully makes the diff a lot better 🚀

Reply:
Added in https://github.com/metabase/metabase/pull/49251/commits/1c74368a069656251ea9f737514142cb2d2a8634

By the way, I think we should actually add a separate section for static and interactive question props. I might do that in a later PR.

Reply:
Yeah I figure, maybe we should have a section where the props apply to all question components and maybe props that only exist for a specific component to avoid confusion. Now we're just throwing everything in the same table.